strings/sillytips.txt

```diff
@@ -28,4 +28,6 @@ Just the tip?
 Some people are unable to read text on a game where half of it is based on text.
 As the Captain, you can use a whetstone to sharpen your fancy fountain pen for extra robustness.
 As the Lawyer, you are the last bastion of roleplay-focused jobs. Even the curator got a whip to go fight people with, that sellout!
-There are at least 11 ways to get through plastic flaps. How many can you name?
\ No newline at end of file
+There are at least 11 ways to get through plastic flaps. How many can you name?
+FEED ME A STRAY CAT
```

Comment:
if we're going to add memes can we actually put "never pet a burning dog"/"don't spit into the wind" in

Reply:
Huh, I must have overlooked that line when squashing commits. Lemme remove tomorrow

Reply:
wait never mind this is sillytips